src/frontend/src/components/DroneOperatorTask/DescriptionSection/UploadsBox/index.tsx

```diff
@@ -13,12 +13,25 @@ const UploadsBox = ({
 }) => {
   const dispatch = useTypedDispatch();
   const files = useTypedSelector(state => state.droneOperatorTask.files);
-  const handleFileChange = (event: any) => {
+  const handleFileChange = async (event: any) => {
```

Comment:
Try avoiding the use of any type. We should be hard and fast about using proper type.